packages/cmr-client/src/ingestConcept.ts

```diff
@@ -25,27 +23,33 @@ const logDetails = {
  * @param {Object} headers - the CMR headers
  * @returns {Promise.<Object>} the CMR response object
  */
-async function ingestConcept(type, xmlString, identifierPath, provider, headers) {
-  let xmlObject = await parseXMLString(xmlString);
-
-  const identifier = property(identifierPath)(xmlObject);
+async function ingestConcept(
+  type: string,
+  xmlString: string,
+  identifierPath: string,
+  provider: string,
+  headers: Headers
+) {
+  const xmlObject = await parseXMLString(xmlString);
+
+  const identifier = <string>property(identifierPath)(xmlObject);
   logDetails.granuleId = identifier;
 
   try {
     await validate(type, xmlString, identifier, provider);
 
     const response = await got.put(
-      `${getUrl('ingest', provider)}${type}s/${identifier}`,
+      `${getIngestUrl({ provider })}${type}s/${identifier}`,
       {
         body: xmlString,
         headers,
       }
     );
 
-    xmlObject = await parseXMLString(response.body);
+    const xmlErrorObject = <{errors?: {error: string}}>(await parseXMLString(response.body));
```

Comment:
It's not necessarily an error object unless it has `errors`? `xmlObject` seems like a better name

Reply:
ea9797eb0